### DISCUSSION TITLE
Identity provider mapper misbehavior in v19 UI

### DISCUSSION THREAD

**Discussion by eggyknap** [keycloak/keycloak#13895] [Archive]
I upgraded recently from v18 to v19, and several of my "hardcoded attribute" identity provider mappers have needed to be rebuilt. When I'm in the v19 console and open a mapper I created in v18, it gives me an error message about parsing JSON, and forces me to reload the admin console. I'm not sure this behavior is limited to "hardcoded attribute" mappers. When I create a new one in v19, I fill out the "user attribute" value with a string name as usual, and when I open the mapper again to view it, I see "[object Object]" in the "User Attribute" field, rather than the attribute name I configured. I haven't yet been able to test that this mapper actually works.
![Selection_205](https://user-images.githubusercontent.com/59133/185670286-b4d91c0d-436b-45a6-929d-7dc217fd0a89.png)

**Comment by rmoqpd5t** [comment]
I noticed mapper-related JSON parsing issues as well and opened https://github.com/keycloak/keycloak/issues/13904

I use hardcoded attributes in each Identity Provider.  However, the JSON error appears clicking on ANY attribute within the Identity Provider.
